tools/total-contributions

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import pathlib
+import subprocess
+import sys
+from datetime import datetime
+from typing import Dict, List
+
+
+def add_log(committer_dict: Dict[str, int], input: List[str]) -> None:
+    for dataset in input:
+        committer_name = dataset.split("\t")[1]
+        commit_count = int(dataset.split("\t")[0])
+        if committer_name in committer_dict:
+            committer_dict[committer_name] += commit_count
+        else:
+            committer_dict[committer_name] = commit_count
```

Comment:
nit: can simplify as
```suggestion
        committer_dict[committer_name] = commit_count + committer_dict.get(committer_name, 0)
```

Or use a `defaultdict(int)` instead of a dict, and then just `+=` will do it.

Reply:
Yeah, was meaning to do this if I had time to cleanup, applied.

Reply:
This version now has a small latent bug:
```py
def add_log(committer_dict: Dict[str, int], input: List[str]) -> None:
    for dataset in input:
        committer_name = dataset.split("\t")[1]
        commit_count = int(dataset.split("\t")[0])
        committer_dict[committer_name] += commit_count
```

in that the types say it accepts any old dict of `str` keys and `int` values, but in fact it relies on its caller passing it specifically a defaultdict.